src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -469,4 +453,42 @@ private String evaluateMacro(String template) {
             throw new IllegalArgumentException(e);
         }
     }
+
+    private void pullRequestActionable(List<PotentialAction> paList, List<Fact> factsList) {
+        Job job = run.getParent();
+        ItemGroup parent = job.getParent();
+        if (parent instanceof MultiBranchProject) {
+            BranchProjectFactory projectFactory = ((MultiBranchProject) parent).getProjectFactory();
+            if (projectFactory.isProject(job)) {
+                Branch branch = projectFactory.getBranch(job);
+                SCMHead head = branch.getHead();
+
+                if (head instanceof ChangeRequestSCMHead) {
+                    String pronoun = StringUtils.defaultIfBlank(head.getPronoun(), "Change Request");
```

Comment:
When ran together with Bitbucket :(
https://github.com/jenkinsci/bitbucket-branch-source-plugin/pull/70
![image](https://user-images.githubusercontent.com/1661688/31313990-1f7fe150-abf4-11e7-89eb-13191139e54b.png)

When used with this comment, we get it nicely
![image](https://user-images.githubusercontent.com/1661688/31313997-5d904aa2-abf4-11e7-8129-638bba8d96b0.png)
Though numbers seem hard 👍